projects/common/models/src/person.model.ts

```diff
@@ -1,8 +1,40 @@
 import * as moment_ from 'moment';
 import { Base } from './base';
 import { SimpleDate } from './simple-date.interface';
+import { Address } from './address.model';
+
 const moment = moment_;
 
+export enum Gender {
+  Female = <any>'F',
+  Male = <any>'M'
+}
+
+
+/**
+ * Various statuses in Canada
+ */
+export enum StatusInCanada {
```

Comment:
All of these changes should be in MSP for now - not the library.  The library isn't meant to have every single data type each application will need.

The idea is that in MSP you would extend this Person model and add the additional attributes you need.

```class MspPerson extends Person { ... } // or rename as appropriate, 'User', etc. ```

Because all other applications, FPC, PRIME, SiteReg and more, do not care about things like StatusInCanada, Activities, movedFromProvince, etc.

I think the simplest approach is to simply roll-back all your changes to the Person model, and and instead to copy those changes to a new MSP class that extends from Person.